### ISSUE TITLE
Explore deploying a single hub with individual groups

### ISSUE THREAD

**Issue by kabilar** [dandi/dandi-hub#184]
1. For the DANDI, LINC, and BICAN projects we are running 3 separate JupyterHubs.
2. Running a separate hub for each project potentially increases the total base costs compared to running a single hub with separate groups.
3. Let's explore what it would take to run all projects on the same hub, but separate the profiles (and potentially billing) using groups.

**Comment by asmacdo** [comment]
I agree that the base cost of running all three as a single hub would be a bit less expensive, maybe $10/day/hub or so base cost. 

Here's the challenges I anticipate:
 - cleanly separating the costs: If the math can be a little bit fuzzy, not so much of an issue.
 - custom auth: I am not sure if groups can have a separate auth mechanism, which will be especially challenging to have custom auth endpoint for dandihub and linc, but GitHub org based for bican.

**Comment by kabilar** [comment]
Thanks for breaking it down, Austin.  Let's discuss more over the next few weeks.

**Comment by kabilar** [comment]
Done
